modules/csg/csg_shape.cpp

```diff
@@ -1745,9 +1745,10 @@ CSGBrush *CSGPolygon3D::_build_brush() {
 			break;
 		case MODE_PATH: {
 			extrusions = Math::ceil(1.0 * curve->get_point_count() / path_interval);
-			if (!path_joined) {
+			if (path_joined) {
+				extrusions += 1;
+			} else {
 				end_count = 2;
-				extrusions -= 1;
```

Comment:
Why did you change this part?  If the path_interval is 1, you should just have 1 segment, correct?  But there are 2 curve points, so you'll get a subdivision even if the interval is set to 1.

(side note: I'm not sure if using the number of points on the curve makes sense at all, as the subdivisions won't match up 1:1 with the curve points).

Reply:
I thought it was undercounting, but it isn't. I'll change it back.